src/frontendFixes/Fixes/metaViewportScalableFix.js

```diff
@@ -0,0 +1,29 @@
+const MetaViewportScalable = window.edac_frontend_fixes.meta_viewport_scalable || {
+	enabled: false,
+};
+
+const MetaViewportScalableFix = () => {
+	if ( ! MetaViewportScalable.enabled ) {
+		return;
+	}
+
+	// Get the meta viewport tag.
+	const metaViewport = document.querySelector( 'meta[name="viewport"]' );
+	if ( metaViewport ) {
+		// check if it has scalable set to no.
+		if ( metaViewport.content.includes( 'user-scalable=no' ) ) {
+			// Remove the meta viewport tag to allow it to be recreated.
```

Comment:
Isn't this missing?

`metaViewport.remove();`

Reply:
Yes, this was missing. I refactored this to add it and avoid an if..else